lib/testpool/single-process-test-pool.js

```diff
@@ -0,0 +1,135 @@
+const fs = require('fs');
+const Bluebird = require('bluebird');
+const debug = require('debug')('ava:pool');
+const {failTypes} = require('../fork');
+const precompilerHook = require('../worker/precompiler-hook');
+const TestPool = require('./test-pool');
+const isolatedRequire = require('./isolated-require');
+
+const preTestPath = require.resolve('./test-pre-require.js');
+const postTestPath = require.resolve('./test-post-require.js');
+
+const preTestContent = fs.readFileSync(preTestPath, 'utf8').toString();
+const postTestContent = fs.readFileSync(postTestPath, 'utf8').toString();
```

Comment:
Since you provided the `'utf8'` encoding `readFileSync()` already returns strings.

Reply:
This ended up getting removed, but good to know.